src/prefabs/structures/SingleValueAutocompleteInput/options/index.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  option,
+  variable,
+  toggle,
+  buttongroup,
+} from '@betty-blocks/component-sdk';
+import { validation } from './validation';
+import { styles } from './styles';
+import { advanced } from './advanced';
+
+export const options = {
+  actionProperty: option('ACTION_JS_PROPERTY', {
+    label: 'Property',
+    value: '',
+  }),
+
+  ...validation,
```

Comment:
I think validations come after the label and value

https://bettyblocks.atlassian.net/wiki/spaces/PT/pages/3166142496/Input+-+Options+Ordering